.github/workflows/qa.yaml

```diff
@@ -15,6 +15,7 @@ env:
     libglib2.0-dev
     libpwquality-dev
 
+  # Drop ffmpeg when https://github.com/charmbracelet/vhs/pull/591 is released.
```

Comment:
This comment is obsolete

Reply:
Yeah, it gets removed in the next commit... I like to have things preserving the state of the current situation.

The comment is moved below though because still we should remove the workaround once that merges.